requirements_file input fails with ansible-galaxy: command not found when using uv-managed install

<!--- Verify first that your issue is not already reported on GitHub -->
<!--- Also test if the latest release and main branch are affected too -->
##### Summary
When using the `requirements_file` input in the GitHub Action, the action fails with `ansible-galaxy: command not found` (exit code 127). The action installs `ansible-core` via uv into an isolated virtual environment but never adds that environment's `bin/` directory to `$GITHUB_PATH`, so the subsequent `ansible-galaxy` shell step cannot find the binary.

##### Issue Type
- Bug Report

##### OS / ENVIRONMENT
<!--- Paste verbatim output between triple backticks -->
```console (paste below)
ansible-lint 26.4.0 using ansible-core:2.20.5 ansible-compat:26.3.0 ruamel-yaml:0.19.1 ruamel-yaml-clib:0.2.15
```

- ansible installation method: uv (managed by the action)
- ansible-lint installation method: GitHub Action (`ansible/ansible-lint@v26.4.0`)
- Runner OS: ubuntu-24.04
- uv version: 0.11.8

##### STEPS TO REPRODUCE
```console (paste below)
- name: Run ansible-lint
  uses: ansible/ansible-lint@v26.4.0
  with:
    setup_python: "true"
    python_version: "3.12"
    requirements_file: "./requirements.yml"
```

##### Desired Behavior
`ansible-galaxy install -r ./requirements.yml` runs successfully using the `ansible-galaxy` binary installed alongside `ansible-core` by uv.

##### Actual Behavior
```paste below
ansible-lint 26.4.0 using ansible-core:2.20.5 ansible-compat:26.3.0 ruamel-yaml:0.19.1 ruamel-yaml-clib:0.2.15
Run ansible-galaxy install -r ./requirements.yml
  ansible-galaxy install -r ./requirements.yml
  shell: /usr/bin/bash --noprofile --norc -e -o pipefail {0}
  env:
    ACTION_REF: 5fac056c45595896c973fbde871f01f6cb14d74c
    UV_PYTHON_INSTALL_DIR: /home/runner/.local/share/uv/python
    UV_PYTHON: 3.12
    UV_CACHE_DIR: /home/runner/.cache/uv
ansible-galaxy: command not found
Error: Process completed with exit code 127.
```

The uv venv `bin/` directory containing `ansible-galaxy` is never exported to `$GITHUB_PATH` after the uv install completes, causing the subsequent shell step to be unable to find the binary. The fix would be to add that path to `$GITHUB_PATH` after the uv install.

--- a/action.yml
+++ b/action.yml
@@ -101,7 +101,8 @@ runs:
       # SETUPTOOLS_SCM_PRETEND_VERSION=${{ github.action_ref || 'main' }}
       run: |
         if [[ "${{ inputs.setup_python }}" == "true" ]]; then
-          uv tool install --python ${{ inputs.python_version }} "ansible-lint[lock] @ git+https://github.com/ansible/ansible-lint@$GH_ACTION_REF"
+          # --with-executables-from requires uv >= 0.8.5 (provided by setup-uv latest above)
+          uv tool install --python ${{ inputs.python_version }} --with-executables-from ansible-core "ansible-lint[lock] @ git+https://github.com/ansible/ansible-lint@$GH_ACTION_REF"
         else
           echo "setup_python is false, using pip"
           pip install "ansible-lint[lock] @ git+https://github.com/ansible/ansible-lint@$GH_ACTION_REF"

